src/Bridge/Symfony/Validator/Validator.php

```diff
@@ -26,7 +26,7 @@
  *
  * @final
  */
-class Validator implements ValidatorInterface
+class Validator implements ConstraintValidatorInterface
```

Comment:
We can't change an interface, the backward compatible solution is to add another interface that extends the first one. We did this with the ContextAwareDataProviderInterface

Reply:
Same response as above. As `ConstraintValidatorInterface` is extending `ValidatorInterface`, it should not cause any BC issues. WDYT?